generators/app/templates/webapp/controller/BaseController.ts

```diff
@@ -31,7 +31,7 @@ export default abstract class BaseController extends Controller {
 	 * Convenience method for getting the i18n resource bundle of the component.
 	 * @returns The i18n resource bundle of the component
 	 */
-	public getResourceBundle(): ResourceBundle | Promise<ResourceBundle> {
+	public getResourceBundle(): Promise<ResourceBundle> {
```

Comment:
as discussed: this likely leads to an error, as the type definitions say:
`getResourceBundle(): ResourceBundle | Promise<ResourceBundle>;`

Reply:
Okay, as the ResourceModel is configured as async in manifest.json, this is true.